src/data/set/lattice.lean

```diff
@@ -1526,6 +1526,18 @@ lemma disjoint_iff_subset_compl_left :
   disjoint s t ↔ t ⊆ sᶜ :=
 disjoint_right
 
+lemma _root_.disjoint.image {s t u : set α} {f : α → β} (h : disjoint s t) (hf : inj_on f u)
```

Comment:
I wonder if this can be recovered from `disjoint.map_order_iso` somehow, not that it really matters.

Reply:
What about `inj_on.image_inter` (does not exist yet)? Otherwise LGTM.
bors d+